.github/workflows/style.yaml

```diff
@@ -19,6 +19,8 @@ jobs:
       uses: actions/setup-python@v4.5.0
       with:
         python-version: '3.10'
+        cache: 'pip'
+        cache-dependency-path: 'requirements/style.txt'
```

Comment:
Do we have to do anything to skip the pip install after this?

Reply:
This is fun. It looks like `cache` only caches downloads, not installs. Compared to develop, avoiding these downloads shaves 25 secs off the "Install pip dependencies" step, but adds 35 secs to the "Set up python" step because it has to download and unpack the cache.